src/modules/hide_hypechat/index.js

```diff
@@ -0,0 +1,17 @@
+import {ChatFlags, PlatformTypes, SettingIds} from '../../constants.js';
+import settings from '../../settings.js';
+import {hasFlag} from '../../utils/flags.js';
+import {loadModuleForPlatforms} from '../../utils/modules.js';
+
+class HideHypeChatModule {
+  constructor() {
+    settings.on(`changed.${SettingIds.CHAT}`, () => this.load());
+    this.load();
+  }
+
+  load() {
+    document.body.classList.toggle('bttv-hide-hypechat', !hasFlag(settings.get(SettingIds.CHAT), ChatFlags.HYPECHAT));
```

Comment:
Some channels and pages don't have this button, I'd query for it and ensure it's there.

And then apply a style to the button, if it's set to be hidden.